samples/apps/copilot-chat-app/README.md

```diff
@@ -59,9 +62,7 @@ First, let’s set up and verify the back-end API server is running.
 1. Build and run the back-end API server
     1. Open a terminal and navigate to `samples/apps/copilot-chat-app/webapi`
     
-    1. Run `dotnet build` to build the project.
-    
-    1. Run `dotnet run` to start the server.
+    1. Execute `dotnet run` to build and start the server.
```

Comment:
Now that I read this rendered, we should break any commands that we expect users to run into full code sections. For example, instead of `dotnet run`, we should do
```
dotnet run
```

This should make it obvious which commands need to be run and easier to copy/paste.

Reply:
Changed.  Much better.